ert_gui/simulation/run_dialog.py

```diff
@@ -92,6 +93,8 @@ def __init__(self, config_file, run_model, simulation_arguments, parent=None):
 
         self._job_view = QTableView(self)
         self._job_view.horizontalHeader().setSectionResizeMode(QHeaderView.Stretch)
+        self._job_view.clicked.connect(self._job_clicked)
+        self._open_files = {}
```

Comment:
It could be that `_job_view.clicked` is a bit too low level for the `RunDialog`.

One alternative is to have the job view emit some `file_selected` event with the specific file.

Easier to unit test this stuff, since it avoids instantiation of the `RunDialog`.